doc/pages/keys.asciidoc

```diff
@@ -331,8 +325,8 @@ the Shift modifier and moving will extend the selection instead.
     move forward in history
 
 *&*::
-    align selection, align the cursor of selections by inserting spaces
-    before the first character of the selection
+    align selections, align the cursor of selections by inserting spaces
```

Comment:
In other places we've used "end of the selection" rather than "cursor", so I guess this should be "...align the end of each selection..."